src/core/public/styles/chrome/_project.scss

```diff
@@ -0,0 +1,16 @@
+.kbnBody--projectLayout {
+  .header__actionMenu {
+    /* fixates the elements position in the viewport, removes the element from the flow of the page */
+    position: fixed;
+    /* position below the primary fixed EuiHeader in the viewport */
+    top: $euiHeaderHeightCompensation;
+    height: $euiHeaderHeightCompensation;
+    /* use a high z-index since this component is part of the header */
+    z-index: $euiZHeader;
+  }
```

Comment:
Ideally I'd prefer these styles to live in CSS-in-JS as opposed to Sass, but I know EUI hasn't made `$euiHeaderHeightCompensation` easily available yet in JS. FYI that the (hopefully near future) plan is to set a global CSS variable for the current header(s) height and allow consumers to access the offset through that.